### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Norgh1**

### CODE REVIEW

File: Chat-App/ViewControllers/OnboardingViewController.swift [hunk 1]
```diff
@@ -9,10 +9,55 @@ import UIKit
...
+    @IBOutlet weak var slider: SliderView!
+    
+    
+    
+    
     override func viewDidLoad() {
         super.viewDidLoad()
+        //TODO: create UI
+        
+        self.view.backgroundColor = .systemBlue
```
Comment (aslanyanhaik) [inline Chat-App/ViewControllers/OnboardingViewController.swift]: why this is not done in storyboard?

File: Chat-App/ViewControllers/OnboardingViewController.swift [hunk 2]
```diff
@@ -9,10 +9,55 @@ import UIKit
 
 final class OnboardingViewController: UIViewController {
 
+    @IBOutlet weak var lionLabel: UILabel!
```
Comment (aslanyanhaik) [inline Chat-App/ViewControllers/OnboardingViewController.swift]: do we need `@IBoutlets`? we can use view `viewWithTag` method. Can't we?

File: Chat-App/ViewControllers/OnboardingViewController.swift [hunk 3]
```diff
@@ -9,10 +9,55 @@ import UIKit
...
+    
     override func viewDidLoad() {
         super.viewDidLoad()
+        //TODO: create UI
+        
+        self.view.backgroundColor = .systemBlue
+        self.sliderView.layer.cornerRadius = 35
+        self.slider.layer.cornerRadius = 35
+        rotateEmojis()
+        setConstraintsEmojis()
```
Comment (aslanyanhaik) [inline Chat-App/ViewControllers/OnboardingViewController.swift]: Constrains can be added in storyboard

File: Chat-App/ViewControllers/OnboardingViewController.swift [hunk 4]
```diff
@@ -9,10 +9,55 @@ import UIKit
...
+    
+    
+    
+    
     override func viewDidLoad() {
         super.viewDidLoad()
+        //TODO: create UI
+        
+        self.view.backgroundColor = .systemBlue
+        self.sliderView.layer.cornerRadius = 35
```
Comment (aslanyanhaik) [inline Chat-App/ViewControllers/OnboardingViewController.swift]: why we use `self` keyword?

Comment (aslanyanhaik) [inline Chat-App/ViewControllers/OnboardingViewController.swift]: We can write an `IBInspecable` properties as extensions to view, to set corner radius in storyboard

File: Chat-App/ViewControllers/OnboardingViewController.swift (hunk 1 above)
Comment (Norgh1) [reply Chat-App/ViewControllers/OnboardingViewController.swift]: idk really, fixed already

File: Chat-App/ViewControllers/OnboardingViewController.swift (hunk 2 above)
Comment (Norgh1) [reply Chat-App/ViewControllers/OnboardingViewController.swift]: offcourse, done!

File: Chat-App/ViewControllers/OnboardingViewController.swift (hunk 3 above)
Comment (Norgh1) [reply Chat-App/ViewControllers/OnboardingViewController.swift]: fixed as wall.

File: Chat-App/ViewControllers/OnboardingViewController.swift (hunk 4 above)
Comment (Norgh1) [reply Chat-App/ViewControllers/OnboardingViewController.swift]: fixed!

Comment (Norgh1) [reply Chat-App/ViewControllers/OnboardingViewController.swift]: Thank you for replay! fixed

**Pull request merged**